velox/functions/CoverageUtil.h

```diff
@@ -17,9 +17,19 @@
 
 #include <string>
 #include <unordered_set>
+#include <vector>
 
 namespace facebook::velox::functions {
 
+// Returns sorted list of scalar function names available in Velox.
+std::vector<std::string> getSortedScalarNames();
```

Comment:
I know these are part of the PR for later but how are they used here? I don't see the implementation.

Reply:
Thanks for pointing it out, yes these are part of a separate PR for later, I added it here as this was a draft PR and will move it to separate PR subsequently.

Reply:
@czentgr : These are exisiting functions. I think Pramod is only making them visible externally in the header.